Makefile

```diff
@@ -13,8 +13,9 @@ test-packages:
 	./tests/packages.sh
 
 test-release:
+	@if [ ! -f "$$GOPATH/bin/goreleaser" ]; then echo "Error: goreleaser is not installed\n\nYou can install goreleaser with 'go install github.com/goreleaser/goreleaser@latest'" && exit 1; fi
 	doppler run -- goreleaser release --snapshot --skip-publish --rm-dist --parallelism=4
 
 scan:
-	if [ ! -f "$$GOPATH/bin/gosec" ]; then echo "Error: gosec is not installed\n\nYou can install gosec with 'go get github.com/securego/gosec/cmd/gosec'\n" && exit 1; fi
-	$$GOPATH/bin/gosec -quiet ./pkg/...
```

Comment:
Why remove the `GOPATH` part?

Reply:
Good catch @nmanoogian! I updated the go install commands and was testing the commands locally and they didn't work so I had assumed something was up with the `GOPATH` but that was my local setup, so these changes are not adequate.

Re-adding the `$$GOPATH/bin` part